### PR TITLE
fix

### PR DESCRIPTION
**Pull request by korsa719**
replace find -> some
press Enter send action create post on the post creator form

### CODE REVIEW

File: client/src/components/Post.js [hunk 1]
```diff
@@ -37,7 +39,7 @@ class Post extends Component {
           {this.props.post.content}
         </p>
         <footer>
-          <a className={favorites.find(post => post.id === this.props.post.id) === undefined ? 'like' : 'likes'}
+          <a className={favorites.some(post => post.id === this.props.post.id) === false ? 'like' : 'likes'}
```
Comment (zozich) [inline client/src/components/Post.js]: You don't need boolean comparison here

File: client/src/components/Feed.js [hunk 2]
```diff
@@ -1,33 +1,52 @@
...
     this.state = {
       text: ''
     }
   }
-
   componentDidMount () {
     if (this.props.posts.length === 0) {
       this.props.loadPosts()
     }
+    if (this.props.favorites.length === 0) {
```
Comment (zozich) [inline client/src/components/Feed.js]: Use destructuring everywhere instead of duplicating this.props.

**Comment by korsa719** [comment]
исправил

File: client/src/components/Post.js (hunk 1 above)
Comment (korsa719) [reply client/src/components/Post.js]: согласен исправил

**Pull request merged**